content/gotraining.slide

```diff
@@ -425,6 +429,53 @@ As part of all labs
 - Create a PR following the [[https://github.com/anz-bank/go-training#pre-pr-checklist][PR checklist]]
 
 
+* PR from own fork - initial setup
+
+Issue PRs from `feature` branch of your own fork to `master` on upstream:
+
+- Login to github.com
+- Click `Fork` in the top right corner of [[https://github.com/anz-bank/go-training-slides]]
+- Set up the following remotes (`USERNAME` is your GitHub username):
+
+	language:none
+	git remote -v
+		origin	https://github.com/USERNAME/go-training-slides.git (fetch)
+		origin	https://github.com/USERNAME/go-training-slides.git (push)
+		upstream	https://github.com/anz-bank/go-training-slides.git (fetch)
+		upstream	https://github.com/anz-bank/go-training-slides.git (push)
+
+- Learn more about remotes and modify them with:
+
+	language:none
+	git help remote
+	git remote add
+	git remote rename
+	git remote delete
+
+
+* PR from own fork - typical git workflow
+
+Start work on a new branch `lab1` with:
+
+	git checkout master
+	git pull upstream master
```

Comment:
This can be shortened to `git pull` if you setup master to track upstream as done in the `git branch` command I added earlier. This should be standard that master should track upstream (IMHO, not sure if universal).

Although on reflection it might be better to keep this explicit.